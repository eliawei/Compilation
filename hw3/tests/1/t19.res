---end scope---
---end scope---
---end scope---
---end scope---
---end scope---
---end scope---
x BYTE -1
i BYTE 0
---end scope---
---end scope---
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
f (BYTE)->BOOL 0
g ()->BYTE 0
main ()->VOID 0
