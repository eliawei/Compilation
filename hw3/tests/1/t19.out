---end scope---
---end scope---
---end scope---
---end scope---
---end scope---
---end scope---
x BYTE -1
i BYTE 0
---end scope---
line 17: type mismatch
